### ISSUE TITLE
Warnings in current release from new changes

### ISSUE THREAD

**Issue by ethanwhite** [weecology/retriever#1174]
```
ethan@omar:~$ retriever --version
Failed to load script: globi_interaction (/home/ethan/.retriever/scripts/)
Exception: name 'gz' is not defined 
Failed to load script: harvard_forests (/home/ethan/.retriever/scripts/)
Exception: invalid syntax (harvard_forests.py, line 14) 
Failed to load script: globi_interaction (/home/ethan/.retriever/scripts/)
Exception: name 'gz' is not defined 
Failed to load script: harvard_forests (/home/ethan/.retriever/scripts/)
Exception: invalid syntax (harvard_forests.py, line 14) 
Failed to load script: globi_interaction (/home/ethan/.retriever/scripts/)
Exception: name 'gz' is not defined 
Failed to load script: harvard_forests (/home/ethan/.retriever/scripts/)
Exception: invalid syntax (harvard_forests.py, line 14) 
v2.1.0
```

Will bumping the minimum version to `2.1.dev` fix this?

**Comment by henrykironde** [comment]
Have you tried removing retriever reset scripts.

**Comment by ethanwhite** [comment]
I did. Are you not seeing the same thing with 2.1.0?

```
ethan@omar:~$ retriever reset scripts 
Failed to load script: globi_interaction (/home/ethan/.retriever/scripts/)
Exception: name 'gz' is not defined 
Failed to load script: harvard_forests (/home/ethan/.retriever/scripts/)
Exception: invalid syntax (harvard_forests.py, line 14) 
Failed to load script: globi_interaction (/home/ethan/.retriever/scripts/)
Exception: name 'gz' is not defined 
Failed to load script: harvard_forests (/home/ethan/.retriever/scripts/)
Exception: invalid syntax (harvard_forests.py, line 14) 
Failed to load script: globi_interaction (/home/ethan/.retriever/scripts/)
Exception: name 'gz' is not defined 
Failed to load script: harvard_forests (/home/ethan/.retriever/scripts/)
Exception: invalid syntax (harvard_forests.py, line 14) 

This will remove existing scripts. 
Specifically it will remove the scripts folder in /home/ethan/.retriever.
Do you want to proceed? (y/N)
y
ethan@omar:~$ retriever --version
Downloading scripts...
Download Progress: [#######################################################################################################################################################################################################################################################] 100.00%
The retriever is up-to-date
Failed to load script: globi_interaction (/home/ethan/.retriever/scripts/)
Exception: name 'gz' is not defined 
Failed to load script: harvard_forests (/home/ethan/.retriever/scripts/)
Exception: invalid syntax (harvard_forests.py, line 14) 
v2.1.0
```

**Comment by ethanwhite** [comment]
Fixed in #1176.
